content/applications/hr/lunch/management.rst

```diff
@@ -0,0 +1,189 @@
+
+.. _lunch/confirm-orders:
+
+Confirm Orders
+--------------
+
+After orders have been sent to the vendor, the next step is to confirm the orders after they have
+been delivered.
+
+On the :guilabel:`Today's Orders` dashboard, click the :guilabel:`Confirm Orders` button that
+appears next to the vendor's name and phone number. Once confirmed, the :guilabel:`Confirm Orders`
+button disappears, and the :guilabel:`Status` column is updated from blue :guilabel:`Sent` tags to
+green :guilabel:`Received` tags, indicating the vendor has delivered the orders. In addition, the
+:guilabel:`✖️ Cancel` icon at the end of each product line changes to a :guilabel:`✉️ Send
+Notification` icon.
+
+If needed, instead of confirming all of the individual products from a vendor, individual products
+can be marked as confirmed. To confirm an individual product, click the :guilabel:`✔️ Confirm` icon
+at the end of the individual product line. When confirming individual products with this method, the
+:guilabel:`Confirm Orders` button remains on the vendor line.
+
+.. image:: management/confirm.png
+   :align: center
+   :alt: The Today's Orders dashboard, with the two different ways to confirm an order highlighted.
+
+.. example::
+   A vendor receives an order for three pizzas and an order of garlic knots. When the delivery is
+   made to the company, the *Lunch* manager notices the garlic knots are missing.
+
+   The manager first marks the three pizzas as received by individually confirming the products with
+   the :guilabel:`✔️ Confirm` icon at the end of each product line.
+
+   Later, when the vendor delivers the garlic knots, the manager can either click the :guilabel:`✔️
+   Confirm` icon at the end of the line for the garlic knots, or click the :guilabel:`Confirm
+   Orders` button that appears next to the vendor's name and phone number.
+
+.. _lunch/notify:
+
+Notify employees
+----------------
+
+After products are received and the orders are confirmed, the employees must be informed that their
+orders have been delivered and are ready to be picked up.
+
+Unlike sending and confirming orders, notifications must be sent individually and cannot be sent in
+a batch.
+
+To notify the user their product has arrived, click the :guilabel:`✉️ Send Notification` icon at the
+end of each product line. An email is sent to the user informing them their products have been
+delivered.
+
+.. important::
+   It is important for each individual company to configure their day-to-day workflow to include
+   the necessary steps for the *Lunch* app. That includes designating people to be responsible for
+   sending and confirming orders, informing employees when orders have arrived, and determining
+   where orders are stored while waiting to be picked up by employees. These are parts of the
+   *Lunch* app flow that are not configured within Odoo and are determined by each individual
+   company.
```

Comment:
I *do* feel like you've mentioned something similar to this in the earlier part of the doc, so you may not actually need this admonition. Just a suggestion...